datasets/fr/assemblee/crawler.py

```diff
@@ -117,7 +117,7 @@ def crawl_acteur(context, data: Dict[str, Any]):
     context.audit_data(birth, ["depNais"])
     dod = ec.pop("dateDeces")
     if is_not_nil(dod):
-        person.set("deathDate", h.parse_date(dod, ["%Y-%m-%d"]))
+        h.apply_date(person, "deathDate", dod)
```

Comment:
we can get rid of is_not_nil using a couple of type.date lookups, right? at least these calls but I think more

Reply:
it gets into the values and double-checks `{'@xmlns:xsi': 'http://www.w3.org/2001/XMLSchema-instance', '@xsi:nil': 'true'}` whether `xsi:nil` is true. I don't think we can solve it with the lookups

Reply:
ah yeah - dict is unhashable, it says.